proto/stream_plan.proto

```diff
@@ -85,6 +85,9 @@ message HashJoinNode {
   // Whether to use delta join for this hash join node.
   // TODO: remove this in the future when we have a separate DeltaHashJoin node.
   bool is_delta_join = 6;
+  // Used for internal table states.
+  plan.TableRefId left_table_ref_id = 7;
```

Comment:
`plan.TableRefId` will be deprecated after Java frontend. We could refactor this with other used places in the future.